doajtest/unit/test_regex.py

```diff
@@ -0,0 +1,18 @@
+from doajtest.helpers import DoajTestCase
+from portality import regex
+
+
+class TestRegex(DoajTestCase):
+
+    def test_http_url(self):
+        assert regex.HTTP_URL_COMPILED.match('http://example.xx')
+        assert regex.HTTP_URL_COMPILED.match('http://example.xx:999')
+        assert regex.HTTP_URL_COMPILED.match('http://example.xx:999/a/b/c')
+        assert regex.HTTP_URL_COMPILED.match('http://example.xx:999/a/b/c?d=e')
+        assert regex.HTTP_URL_COMPILED.match('http://example.xx:999/a/b/c?d=1&f=2#qq')
+        assert regex.HTTP_URL_COMPILED.match('http://example-xxxx.xx:999/a/b/c?d=1&f=2#qq')
+        assert regex.HTTP_URL_COMPILED.match('http://example-xxxx.xx:80/a/b/c?d=1&f=2#qq')
+
+        assert not regex.HTTP_URL_COMPILED.match('http://example.xx:999/a/b/a c?d e=1&f=2#qq')
+        assert not regex.HTTP_URL_COMPILED.match('http://example.xx:999/a/b/c?d e=1&f=2#qq')
+        assert not regex.HTTP_URL_COMPILED.match('http://example.xx:999/a/b/c?  d e=1&f=2#qq')
```

Comment:
Can we move the regexes to a separate fixture file (in the same way as we do with urls:
doajtest/fixtures/urls.py

Reply:
I don't aware `test_regexes` exist. I will remove `test_regex` to avoid duplicate test cases